src/coreclr/tests/src/baseservices/exceptions/stackoverflow/stackoverflow.cs

```diff
@@ -0,0 +1,149 @@
+using System;
```

Comment:
Nit: License headers

Reply:
Oops, thank you for noticing that, fixed.